src/components/Button/src/Button.vue

```diff
@@ -416,4 +489,28 @@ export default {
 .Button.align_space-between .InformationText {
 	margin-right: 8px;
 }
+
+.IconAligner {
+	display: flex;
+	align-items: center;
+	height: 24px;
+	margin-right: 8px;
+}
+
+.Icon {
+	width: 16px;
```

Comment:
Again let's try to use the spacing variables if Maker has them available. If not, we should add them

Reply:
there's an open issue free for anyone to take if they want to tackle CSS reuse within Maker: https://github.com/square/maker/issues/39